docs/examples/tutorial/string/cpp_string.pyx

```diff
@@ -6,7 +6,6 @@ def get_bytes():
     py_bytes_object = b'hello world'
     cdef string s = py_bytes_object
 
-    s.append('abc')
+    s.append(b'abc')
     py_bytes_object = s
     return py_bytes_object
-
```

Comment:
I guess it only fails for you because you are compiling it with `-3`. It works with both `-2` and `--3str`.
Not sure which is better. It's probably ok to explicitly use a byte string here.

Reply:
Fine, then I'll resolve this